easybuild/easyconfigs/s/SqueezeMeta/SqueezeMeta-1.5.0-foss-2021b.eb

```diff
@@ -0,0 +1,80 @@
+easyblock = 'Tarball'
+
+name = 'SqueezeMeta'
+version = '1.5.0'
+
+homepage = 'https://github.com/jtamames/SqueezeMeta'
+description = """SqueezeMeta is a full automatic pipeline for metagenomics/metatranscriptomics,
+ covering all steps of the analysis."""
+
+toolchain = {'name': 'foss', 'version': '2021b'}
+
+source_urls = ['https://github.com/jtamames/SqueezeMeta/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = [
+    'e984d8ad57f859edd7d8ccb26a5a8ec51120f92365c5c00994bc5ea58529d639',  # v1.5.0.tar.gz
+]
+
+dependencies = [
+    ('Perl', '5.34.0'),  # provides XML::Parser
+    ('XML-LibXML', '2.0207'),
+    ('Python', '3.9.6'),
+    ('matplotlib', '3.4.3'),
+    ('Pysam', '0.17.0'),
+    ('R', '4.1.2'),
+    ('BLAST+', '2.12.0'),
+    ('BWA', '0.7.17'),
+    ('CD-HIT', '4.8.1'),
+    ('CheckM', '1.1.3'),
+    ('DAS_Tool', '1.1.1', '-R-%(rver)s'),  # depends on prodigal & pullseq
+    ('DIAMOND', '2.0.13'),
+    ('MEGAHIT', '1.2.9'),
+    ('minimap2', '2.22'),
+    ('Java', '11', '', True),
+    ('RDP-Classifier', '2.13', '-Java-%(javaver)s', True),
+    ('MetaBAT', '2.15'),
+    ('MinPath', '1.6'),  # depends on GPLK (which provides glpsol)
+    ('PRINSEQ', '0.20.4', '-Perl-%(perlver)s'),
+    ('Trimmomatic', '0.39', '-Java-%(javaver)s', True),
+    ('AMOS', '3.1.0'),
+    ('MaxBin', '2.2.7'),
+    ('SPAdes', '3.15.3'),
+    ('Bowtie2', '2.4.4'),
+    ('canu', '2.2'),
+    ('HMMER', '3.3.2'),
+    ('MUMmer', '4.0.0beta2'),
+    ('pplacer', '1.1.alpha19', '', True),
+    ('Short-Pair', '20170125'),
+    ('SAMtools', '1.14'),
+    ('CompareM', '0.1.2'),
+    ('R-bundle-Bioconductor', '3.14', '-R-%(rver)s')
+]
+
+keepsymlinks = True
+
+postinstallcmds = [
+    # backup patched barrnap
+    "cp %(installdir)s/bin/barrnap %(builddir)s/",
+    # remove vendored tools that are include as dependencies
+    "rm -r %(installdir)s/bin/*",
+    "cp %(builddir)s/barrnap %(installdir)s/bin/",
+    # fix shebang in Perl scripts
+    "sed -i 's@#!/usr/bin/perl@#!/usr/bin/env perl@g' %(installdir)s/scripts/*.pl",
+    "mkdir %(installdir)s/R"
```

Comment:
missing comma at the end, so this will result in a directory `Rmkdir` being created (since it's glued to the line below), and that tells me this line is not needed at all?